src/lib/runtime_config/runtime_config.ml

```diff
@@ -1765,18 +1764,13 @@ module Json_loader : Json_loader_intf = struct
     List.filter_opt
       [ config_file_installed; config_file_configdir; config_file_envvar ]
 
-  let load_config_files ?conf_dir ?commit_id_short ~logger config_files =
+  let load_config_files ?conf_dir ?commit_id_short config_files =
     let open Deferred.Or_error.Let_syntax in
     let config_files = List.map ~f:(fun a -> (a, `Must_exist)) config_files in
     let config_files =
       get_magic_config_files ?conf_dir ?commit_id_short () @ config_files
     in
     let%map config_jsons =
-      let config_files_paths =
-        List.map config_files ~f:(fun (config_file, _) -> `String config_file)
-      in
-      [%log info] "Reading configuration files $config_files"
```

Comment:
I agree with @mrmr1993 that just removing the logs is not a good deal.

I wonder, could we, instead of pushing logger into this function, return the result of `load_config_files` all the way up the callstack? Let the function on the top of call chain decide whether it wants to log the info/warn/fatal messages, or just ignore them.